pkg/operator/controller/ttl_report.go

```diff
@@ -63,12 +64,8 @@ func (r *TTLReportReconciler) reconcileReport() reconcile.Func {
 		if err != nil {
 			return ctrl.Result{}, fmt.Errorf("failed parsing %v with value %v %w", v1alpha1.TTLReportAnnotation, ttlReportAnnotationStr, err)
 		}
-		creationTime := report.Report.UpdateTimestamp
-		ttlExpired, durationToTTLExpiration, err := ttlIsExpired(reportTTLTime, creationTime.Time)
-		if err != nil {
-			return ctrl.Result{}, err
-		}
-		if ttlExpired {
+		durationToTTLExpiration := ttlIsExpired(reportTTLTime, report.Report.UpdateTimestamp.Time)
```

Comment:
Do we need these changes in scope of this PR? Maybe we can do refactor PR to limit the "blast radius". There are many changes already to review.

Reply:
made it to avoid duplicate code